src/web/app/components/question-responses/grouped-responses/grouped-responses.component.html

```diff
@@ -2,10 +2,10 @@
     *ngIf="responses.length && responses[0].allResponses.length">
   <div class="col-md-3">
     <div>
-      {{ isGrq ? 'To' : 'From' }}: {{ responses[0].allResponses[0][isGrq ? 'recipient' : 'giver']}}
+      {{ isGrq ? 'To' : 'From' }}: {{ responses[0].allResponses[0][isGrq ? 'recipient' : 'giver']}} {{ isGrq ? responses[0].allResponses[0].recipientTeam != '' ? '(' + responses[0].allResponses[0].recipientTeam + ')' : '' : '(' + responses[0].allResponses[0].giverTeam + ')'}}
```

Comment:
Try this https://stackoverflow.com/a/43710820

Reply:
Yeap I'll take a look, thanks!